tensorflow_datasets/structured/amazon_us_reviews.py

```diff
@@ -108,14 +108,21 @@ def __init__(self, data=None, **kwargs):
 
 class AmazonUSReviews(tfds.core.GeneratorBasedBuilder):
   """AmazonUSReviews dataset."""
-
+  # Version history:
```

Comment:
This file doesn't need to be changed, builder is already using s3 (which is now the default).